modules/rdma-configuring-the-nvidia-network-operator.adoc

```diff
@@ -0,0 +1,230 @@
+            }
+          },
+          {
+            "resourceName": "rdma_shared_device_eth",
+            "rdmaHcaMax": 63,
+            "selectors": {
+              "ifNames": ["ens8f0np0"]
+            }
+          }
+        ]
+      }
+    image: k8s-rdma-shared-dev-plugin
+    repository: ghcr.io/mellanox
+    version: v1.5.1
+  secondaryNetwork:
+    ipoib:
+      image: ipoib-cni
+      repository: ghcr.io/mellanox
+      version: v1.2.0
+  nvIpam:
+    enableWebhook: false
+    image: nvidia-k8s-ipam
+    repository: ghcr.io/mellanox
+    version: v0.2.0
+  ofedDriver:
+    readinessProbe:
+      initialDelaySeconds: 10
+      periodSeconds: 30
+    forcePrecompiled: false
+    terminationGracePeriodSeconds: 300
+    livenessProbe:
+      initialDelaySeconds: 30
+      periodSeconds: 30
+    upgradePolicy:
+      autoUpgrade: true
+      drain:
+        deleteEmptyDir: true
+        enable: true
+        force: true
+        timeoutSeconds: 300
+        podSelector: ''
+      maxParallelUpgrades: 1
+      safeLoad: false
+      waitForCompletion:
+        timeoutSeconds: 0
+    startupProbe:
+      initialDelaySeconds: 10
+      periodSeconds: 20
+    image: doca-driver
+    repository: nvcr.io/nvidia/mellanox
+    version: 24.10-0.7.0.0-0
+    env:
+    - name: UNLOAD_STORAGE_MODULES
+      value: "true"
+    - name: RESTORE_DRIVER_ON_POD_TERMINATION
+      value: "true"
+    - name: CREATE_IFNAMES_UDEV
+      value: "true"
+----
+
+. Create the `NicClusterPolicy` custom resource on the cluster:
++
+[source,terminal]
+----
+$ oc create -f network-sharedrdma-nic-cluster-policy.yaml 
+----
++
+.Example output
+[source,terminal]
+----
+nicclusterpolicy.mellanox.com/nic-cluster-policy created
+----
+
+. Validate the `NicClusterPolicy` by running a few commands in the DOCA/MOFED container:
++
+[source,terminal]
+----
+$ oc get pods -n nvidia-network-operator
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                                          READY   STATUS    RESTARTS   AGE
+doca-telemetry-service-hwj65                                  1/1     Running   2          160m
+kube-ipoib-cni-ds-fsn8g                                       1/1     Running   2          160m
+mofed-rhcos4.16-9b5ddf4c6-ds-ct2h5                            2/2     Running   4          160m
+nic-feature-discovery-ds-dtksz                                1/1     Running   2          160m
+nv-ipam-controller-854585f594-c5jpp                           1/1     Running   2          160m
+nv-ipam-controller-854585f594-xrnp5                           1/1     Running   2          160m
+nv-ipam-node-xqttl                                            1/1     Running   2          160m
+nvidia-network-operator-controller-manager-5798b564cd-5cq99   1/1     Running   2         5d23h
+rdma-shared-dp-ds-p9vvg                                       1/1     Running   0          85m
+----
+
+. You can `rsh` into the `mofed` container to check the status:
```

Comment:
Same comment about writing this in the imperative, using the Optional indicator if applicable